uber/custom_tags.py

```diff
@@ -111,7 +111,8 @@ def yesno(value, arg=None):
 
 @JinjaEnv.jinja_filter
 def jsonize(x):
-    return safe_string('{}' if x is None else html.escape(json.dumps(x, cls=serializer), quote=False))
+    is_empty = x is None or isinstance(x, jinja2.runtime.Undefined)
+    return safe_string('{}' if is_empty else html.escape(json.dumps(x, cls=serializer), quote=False))
```

Comment:
I had to cherry pick this from master in order to test the `registration/register` page.